magefile.go

```diff
@@ -50,5 +50,5 @@ func (Fmt) Check() error {
 // Runs the linter.
 func (Fmt) Lint() error {
 	lintCmd := "golangci-lint"
-	return sh.RunV(lintCmd, "run")
+	return sh.RunV(lintCmd, "run", "--timeout", "2m0s")
```

Comment:
[req]: We use [.golangci.yml](https://github.com/scribd/go-sdk/blob/main/.golangci.yml) to place settings for `golangci-lint` tool. We can update `timeout` setting there as well like we do in [document-search](https://github.com/scribd/document-search/blob/f2fb5c5c672467207f9f19f596d958c44682bb49/.golangci.yml#L3). It would also keep the `magefile` cleaner in this case

Reply:
Oh, I've missed that we have the configuration file for the linter